config.py

```diff
@@ -0,0 +1,111 @@
+import os
+import types
+from functools import reduce
+
+_GLOBAL_CONFIG = dict(
+    # If running on Tools labs, keep database dumps in this directory...
+    archive_dir = os.path.join(os.path.expanduser('~'), 'archives'),
+
+    # ...and delete dumps that are older than this many days
+    archive_duration_days = 90,
+
+    # Where to put various logs
+    log_dir = os.path.join(os.path.expanduser('~'), 'logs'),
+
+    flagged_off = [],
+
+    profile = True,
+
+    stats_max_age_days = 90,
+
+    user_agent = 'citationdetective',
+
+)
+
+_CITATION_NEEDED_CONFIG = dict(
+    # Embeddings for the words in the sentences
+    vocb_path = os.path.expanduser('~/citation-needed/embeddings/word_dict_en.pck'),
+
+    # Embeddings for the section titles
+    section_path = os.path.expanduser('~/citation-needed/embeddings/section_dict_en.pck'),
+
+    # Tensorflow models to detect Citation Need for English
+    model_path = os.path.expanduser('~/citation-needed/models/fa_en_model_rnn_attention_section.h5'),
+
+    # Maximum length of all sequences
+    max_seq_length = 187,
+
+)
+
+# A base configuration that all languages "inherit" from.
+_BASE_LANG_CONFIG = dict(
+
+    statement_max_size = 200,
+
+    context_max_size = 800,
+    
+    hyperlink_regex = "http[s]?://(?:[a-zA-Z]|[0-9]|[$-_@.&+#]|[!*\(\),]|(?:%[0-9a-fA-F][0-9a-fA-F]))+",
+
+    sentence_regex = '(?<!\w\.\w.)(?<![A-Z][a-z]\.)(?<=\.|\?)\s',
```

Comment:
How about we use https://www.nltk.org/api/nltk.tokenize.html#nltk.tokenize.sent_tokenize ?